pkg/controller/context.go

```diff
@@ -98,6 +105,10 @@ type Context struct {
 	// instances for cert-manager.io types
 	SharedInformerFactory informers.SharedInformerFactory
 
+	// MetadataInformerFactory can be used to start partial metadata
+	// informers
+	MetadataInformerFactory metadatainformer.SharedInformerFactory
```

Comment:
Based on the name, it is not clear that this is a Filtered SharedInformerFactory.

Reply:
Thanks, I agree, I've renamed this in https://github.com/cert-manager/cert-manager/pull/5976/commits/11403c821c8375a3ed74039a3a21d2815589a0ba